webpack/ForemanTasks/Components/TasksTable/TasksTable.js

```diff
@@ -101,13 +101,14 @@ const TasksTable = ({
     <div className="tasks-table">
       <Table
         key="tasks-table"
-        columns={createTasksTableSchema(
-          setSortHistory,
-          uriQuery.sort_by,
-          uriQuery.sort_order,
+        columns={createTasksTableSchema({
+          setSort: setSortHistory,
+          by: uriQuery.sort_by,
+          order: uriQuery.sort_order,
           taskActions,
-          getSelectionController()
-        )}
+          selectionController: getSelectionController(),
+          canEdit: results[0].canEdit,
```

Comment:
Does this mean that if the user cannot edit first task in the response, then they won't be able to edit any other task on the same page?

Reply:
This is used for the bulk select all.
Isn't the `edit_foreman_tasks` permission is user based and not task based?

Reply:
Foreman's permission system allows to "scope" the permissions. That means that even though a user has `edit_foreman_tasks`, they may not be able to edit *any* task.

Reply:
I think we should still leave this and notify that some of the selected tasks can't be canceled because the user doesn't have permissions

Reply:
Added Refs #29457 - add canEdit check to bulk actions